cryptography/src/bls12381/benches/bench.rs

```diff
@@ -19,7 +18,6 @@ mod tle_encrypt;
 
 criterion_main!(
     dkg_recovery::benches,
-    dkg_reshare_recovery::benches,
```

Comment:
I kept this because resharing takes much longer than non-reshare recovery? IMHO it is worth keeping...